flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/client/HiveShimV120.java

```diff
@@ -47,6 +51,14 @@
  */
 public class HiveShimV120 extends HiveShimV111 {
 
+	// a set of functions that shouldn't be overridden by HiveModule
+	@VisibleForTesting
+	public static final Set<String> BUILT_IN_FUNC_BLACKLIST = Collections.unmodifiableSet(new HashSet<>(
```

Comment:
should this be in HiveModule rather than hive shim?

Reply:
I put it in the shim so that the filter is closer to the source. But I'm also fine to move it to HiveModule to make things clearer.